PokeOneWeb/Frontend/pokeoneweb/src/app/app-routing.module.ts

```diff
@@ -4,6 +4,10 @@ import { HomeComponent } from './pages/home/home.component';
 import { LoadingComponent } from './pages/loading/loading.component';
 import { NotFoundComponent } from './pages/not-found/not-found.component';
 import { DynamicRouteGuard } from './core/guards/dynamic-route.guard';
+import { FaqComponent } from './pages/faq/faq.component';
+import { AboutComponent } from './pages/about/about.component';
+import { GuidesComponent } from './pages/guides/guides.component';
+import { LocationsComponent } from './pages/locations/locations.component';
```

Comment:
Please follow the naming convention of the other pages and name the component LocationListComponent (location-list.component)

Reply:
Done